dot/utils.go

```diff
@@ -24,8 +24,8 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-// NewTestGenesis returns a test genesis instance using "gssmr" raw data
-func NewTestGenesis(t *testing.T) *genesis.Genesis {
+// newTestGenesis returns a test genesis instance using "gssmr" raw data
+func newTestGenesis(t *testing.T) *genesis.Genesis {
```

Comment:
is it possible to move those not exported helpers test functions into `dot/utils_test.go` file? And move the function `RandomNodeName` into `dot/node.go` and then remove the file `./dot/utils.go`?

Reply:
That's definitely the longer term goal. We should have no test helper exported functions at all (prefer test helping code duplication vs package dependency), so the goal is to (through multiple PRs):

1. Unexport what's unexportable (this PR)
2. Unexport the remaining and duplicate test helper code in packages that need it (btw same goes for mocks in my opinion, we shouldn't share mocks across packages to reduce Go API noise).
3. Rename to utils_test.go (or helpers_test.go)

That 3-steps is to have the minimal amount of deltas too

Reply:
Got it!